docs/migrate_from_masb/mssql.md

```diff
@@ -0,0 +1,67 @@
+# Migrate [Azure SQL Database](https://azure.microsoft.com/en-us/services/sql-database/) Service Instances From MASB To OSBA
+
+***Note***: this guidance is specially for existing server scenario. Straighter, the SQL Databases created by MASB must be on a server whose credentials are provided from MASB manifest.
+
+## Steps
+
+### Create azure-sql-12-0-dbms-registered service instance by OSBA
+
+OSBA doesn't support to provide existing servers' credentials from manifest and create database on them. Instead, it provides **azure-sql-12-0-dbms** service to create SQL servers and you can create database on the server by **azure-sql-12-0-database** service. The service **azure-sql-12-0-dbms-registered** is a service using your existing Azure SQL server. It does NOT CREATE new server in provisioning and does NOT DELETE the server in deprovisioning. You can run the following CF CLI command to create a instance of it:
+
+```
+cf create-service azure-sql-12-0-dbms-registered dbms <instance-name> -c '{"resourceGroup":"<group-name>", "location":"<server-location>", "server":"<server-name>", "administratorLogin":"<login>", "administratorLoginPassword":"<login-password>", "alias":"<instance-name>"}'
+```
+
+### Create azure-sql-12-0-database-from-existing
+
+Similar to the server, the service **azure-sql-12-0-database-from-existing** is a service using your existing Azure SQL database. It does NOT CREATE new server in provisioning but DELETE the server in deprovisioning. Not like the server, it is to take over the database. First, you can run the following command to check the service credentials delivered to you application:
```

Comment:
Should be "It does NOT CREATE new **database** in provisioning but DELETE the **database** in deprovisioning"

Reply:
Good catch! Will fix, thanks.